_posts/2024-05-31-this-month-in-servo.md

```diff
@@ -0,0 +1,455 @@
+---
+layout:     post
+tags:       blog
+title:      "This month in Servo: and more!"
+date:       2024-05-31
+summary:    your text here
+categories:
+---
+
+Servo now supports several CSS features in its nightly builds:
+
+- as of 2024-04-29, **‘start’**, **‘end’**, and **‘space-evenly’** values in **‘align-content’** and **‘justify-content’** (@nicoburns, #31724)
+- as of 2024-04-30, **‘white-space-collapse’**, **‘text-wrap-mode’**, and the new **‘white-space’** shorthand (@Loirooriol, #32146)
+- as of 2024-05-03, **‘ch’** and **‘ic’ font-relative units** (@andreubotella, #32171)
+- as of 2024-05-19, basic support for **‘border-collapse’** (@mrobinson, @Loirooriol, #32309)
+- as of 2024-05-22, **‘empty-cells’** (@Loirooriol, #32331)
+- as of 2024-05-22, **‘visibility: collapse’** on table parts (@Loirooriol, @mrobinson, #32333)
+
+Together with correct sizing for **floating tables** (@Loirooriol, #32150) and **empty list items** (@mrobinson, @Loirooriol, #32152), Servo has made some big strides in the Web Platform Tests this month:
+
+- 90.8% (+1.6pp) in the **CSS2 floats** tests
+- 68.7% (+5.7pp) in the **CSS2 and CSS tables** tests
+- 53.3% (+4.0pp) in the **CSS text** tests
+- 48.8% (+3.3pp) in the **CSS position** tests
+
+We now support **aborting fetch()** with AbortController and AbortSignal (@syvb, #31361), and several DOM properties:
```

Comment:
https://github.com/servo/servo/pull/31361 was reverted in https://github.com/servo/servo/pull/32243.

Reply:
fixed in 5017e403bddcc1c637015ee74b25f57912c361fd